fmn/delivery/backends/base.py

```diff
@@ -0,0 +1,67 @@
+import abc
```

Comment:
I don't see a general copyright file in the repository, so I recommend adding a copyright header to this file.